### PR TITLE
[DOCS-7766] Add OpenContent properties to Enterprise Viewer 4.0.x

### PR DESCRIPTION
**Pull request by mbujak-hyland**

### CODE REVIEW

**Review by adecatur72 — APPROVED** [review]
Looks good!

File: enterprise-viewer/latest/config/files.md [hunk 1]
```diff
@@ -610,3 +610,240 @@ Fail loudly on errors that occur with embedded annotations. This defaults to `tr
...
+
+Whether or not to allow multiple users to make anntations to a document offline and check it in online. If set to `true`, multiple users are allowed to make annotations to a document while not having access to the internet and check in their document once they regain internet access.
+
+Default value: `false`
+
+### annotation.allowExternalReviewers
+
+Whether or not to allow external users, who do not have user accounts, to make annotations to a document. This defaults to `false` so that external users are not allowed to make annotations to a document.
+
+Default value: `true`
```
Comment (anxumalo) [inline enterprise-viewer/latest/config/files.md:636]: Should this be `false` based on the statement above?  "This defaults to `false`

File: enterprise-viewer/latest/config/files.md [hunk 2]
```diff
@@ -610,3 +610,240 @@ Fail loudly on errors that occur with embedded annotations. This defaults to `tr
...
+### annotation.mentionEmailSubjectTemplateName
+
+The filenames of email subject and body templates for offline mentioned emails.
+
+The default filename of email subject template: `mention-email-subject.ftl`
+The default filename of email body template: `mention-email-body.ftl`
+
+### redaction.redactedPageResolution
+
+The resolution used after coverting the redacted document pages into image format (to remove the text).
```
Comment (anxumalo) [inline enterprise-viewer/latest/config/files.md:745]:
```suggestion
The resolution used after converting the redacted document pages into image format (to remove the text).
```

File: enterprise-viewer/latest/config/files.md [hunk 3]
```diff
@@ -610,3 +610,240 @@ Fail loudly on errors that occur with embedded annotations. This defaults to `tr
...
+
+Default value: `72`
+
+### actpdf.index.page.allow.multiline.entries
+
+Whether or not long entries on the index page are shown in full, occupying multiple lines. This defaults to `false` so that each long entry is truncated and fits within a single line.
+If set to `true`, long entries are shown in full, occupying multiple lines.
+
+Default value: `false`
+
```
Comment (anxumalo) [inline enterprise-viewer/latest/config/files.md:801]: I think there are four missing properties with format `ctrlprint.*` - starting from `ctrlprint.lines.per.index.page`.

File: enterprise-viewer/latest/config/files.md [hunk 4]
```diff
@@ -610,3 +610,240 @@ Fail loudly on errors that occur with embedded annotations. This defaults to `tr
...
+### managelegalhold.displayErrorDocProperty
+
+The property of a document that failed to apply legal hold. For example, if set to `objectName`, all the names of documents that failed to apply legal hold are displayed. To list all the IDs of documents that failed to apply legal hold, set it to `objectID`.
+
+Default value: `objectName`
+
+### Path to `FFMpeg` and `FFProbe` executable
+
+The following are paths to the FFMpeg and FFProbe executable:
```
Comment (anxumalo) [inline enterprise-viewer/latest/config/files.md:816]:
```suggestion
### Path to `FFMpeg` and `FFProbe` executables

The following are paths to the FFMpeg and FFProbe executables:
```

**Review by anxumalo — CHANGES_REQUESTED** [review]
I've added some minor suggestions and there may be a few properties missing (unless they're included in another PR)

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**